example.config.yml

```diff
@@ -0,0 +1,47 @@
+default:
+  directories:
+    dir_scenario: "path/to/scenario_folder"
+    dir_abcd: "path/to/abcd_folder"
+    dir_raw: "path/to/raw_folder"
+    dir_matched: "path/to/matched_folder"
+    dir_output: "path/to/output_folder"
+  file_names:
+    filename_raw: "raw_loanbook_123.csv"
+    filename_scenario_tms: "scenarios_2022_tms.csv"
+    filename_scenario_sda: "scenarios_2022_sda.csv"
+    filename_abcd: "abcd.csv"
+  project_parameters:
+    scenario_source: "weo_2022"
+    scenario_select: "nze_2050"
+    region_select: "global"
+    # normally the start year should correspond with year of the publication of
+    # the scenario in use
+    start_year_select: 2022
+    time_frame_select: 5
+    # regions must be available for the selected scenario
+    benchmark_regions_select: "global,european union"
+    remove_inactive_companies: TRUE
+  sector_split:
+    apply_sector_split: TRUE
+    sector_split_type: "equal_weights"
+    # sector_split_type: "worst_case"
+    dir_split_company_id: "path/to/split_folder"
+    filename_split_company_id: "split_company_ids.csv"
+    dir_advanced_company_indicators: "path/to/advanced_company_indicators_folder"
+    filename_advanced_company_indicators: "advanced_company_indicators.xlsx"
+  matching:
+    prep_input_level: "direct_loantaker"
+    params_match_name:
+      by_sector: TRUE
+      min_score: 0.9
+      method: "jw"
+      p: 0.1
+      overwrite: NULL
+      join_id: NULL
+    own_sector_classification:
+      use_own_sector_classification: FALSE
+      dir_own_sector_classification: "path/to/own_sector_classification_folder"
+      filename_own_sector_classification: "own_sector_classification.csv"
```

Comment:
NIT (and unrelated to this PR): maybe rename this to `manual_sector_classification`? 

`own` to me always sounds like `ownership`

Reply:
https://github.com/RMI-PACTA/workflow.multi.loanbook/issues/4